lochness/__init__.py

```diff
@@ -88,15 +88,15 @@ def initialize_metadata(Lochness, args) -> None:
     for study_name in args.studies:
         # if 'redcap' or 'rpms' is in the sources, create (overwrite)
         if 'redcap' in args.input_sources:
-            id_fieldname = 'record_id1'
```

Comment:
In fact, the current master does not have `record_id1`:

https://github.com/AMP-SCZ/lochness/blob/master/lochness/__init__.py#L91

Reply:
Okay. It's anyway updated to `chric_subject_id` for now. It should work. I'll double check in the next PR as well.

Reply:
It should be GitHub's fault. In the head i.e. `master`, Ln 91 has `chric_subject_id`:
https://github.com/AMP-SCZ/lochness/blob/master/lochness/__init__.py#L91

So I don't understand why the diff is being showed!